component/site/assets/js/editicalJQ.js

```diff
@@ -421,24 +421,36 @@ function checkUntil(){
 
 }
 
-function setEndDateWhenNotRepeating(){
+function setEndDateWhenNotRepeating(elem){
+	var id = elem[0].id;
 	var norepeat = document.getElementById("NONE");
 	start_date = document.getElementById("publish_up");
 	end_date = document.getElementById("publish_down");
 
 	startDate = new Date();
-	startDate = startDate.dateFromYMD(start_date.value);	
+	startDate = startDate.dateFromYMD(start_date.value);
+
+	defaultStartDate = new Date();
+    defaultStartDate = startDate.dateFromYMD(start_date.defaultValue);
```

Comment:
where does defaultValue come from ???

Reply:
it is set as part of the Joomla! Calendar